src/Uno.UI/UI/Xaml/Controls/Control/Control.cs

```diff
@@ -146,6 +146,27 @@ protected override void OnLoaded()
 			SetUpdateControlTemplate();
 
 			base.OnLoaded();
+
+			PointerPressed += OnPointerPressed;
+			PointerReleased += OnPointerReleased;
```

Comment:
It's actually not correct to call the virtual method from the event. From the [UWP docs](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.control.onpointerpressed) (emphasis mine):

> Control.OnPointerPressed(PointerRoutedEventArgs) Method
> Called **before** the PointerPressed event occurs.

If the overridden OnPointerPressed method sets Handled=true, then the event won't be raised. That's the reason for all the OnPointerXInternal methods in the existing code. Doing it this way will introduce a regression because the virtual method may be called before or after event handlers.

Reply:
The called method is static and the overriden version is always enlisted as first in the invocation list, so it should be called first too.